### ISSUE TITLE
Review of Spaceflights tutorial for errors

### ISSUE THREAD

**Issue by yetudada** [kedro-org/kedro#1874]
## TL;DR
This task summarises an effort to edit and review the Spaceflights tutorial so that it's error-free for our users.

## Description
Spaceflights is extremely important for onboarding new users of Kedro because it teaches users beginner and intermediate functionality through a staged workflow. You can view our Heap Analytics data to understand the viewership of certain pages within our documentation; you'll see that the tutorial pages are popular. 

![Screenshot 2022-09-26 at 14 51 39](https://user-images.githubusercontent.com/43755008/192294611-3f643700-f483-403b-8f19-296f71c40527.png)

Some of our users have complained that the tutorial has some errors: 

> "Some sort of errors due to some environment setup in the tutorials. That's when they get a bit skeptical about Kedro because the tutorial itself is broken. So how do we get help on that?"

## Proposed action
I would suggest the following: 

- Pinning the version of Kedro that is used in our tutorial. Currently, users are implicitly guided to install the latest version of Kedro from [the Get Started page](https://kedro.readthedocs.io/en/stable/get_started/install.html). [We never actually tell them to install Kedro at all before they run `kedro new`](https://kedro.readthedocs.io/en/stable/tutorial/spaceflights_tutorial.html#kedro-spaceflights-tutorial). This should change to use the exact version of Kedro that we have tested the documentation with.
- A work through of Spaceflights to discover any other errors. 
- An amendment to our release steps to cover bumping the version of Kedro used in our tutorial for major releases; that includes a work through of the tutorial to check for errors

**Comment by datajoely** [comment]
Can I add kedro now works on pyscript #1541 so we can now have a demo which runs purely in the browser (kedro.org) and no environment set up.

**Comment by stichbury** [comment]
I'm happy to pick this up, but cannot assign it to myself

**Comment by stichbury** [comment]
I've found a few issues but they're mostly minor so I can fix those and make some more general revisions to improve readability.

I'm also catching up with @datajoely to talk about the pyscript option to run in the browser, which would be a great thing to include in the docs.

**Comment by stichbury** [comment]
Creating a few sub-issues to deal with things I've found that need -major- revision. I won't document the smaller changes and just adjust the markdown as part of the PR for this issue.
